bin/load-runner.js

```diff
@@ -182,21 +193,35 @@ var l = new loop.MultiLoop({
 
     var scriptArgs = [args.s].concat(args._);
 
-    scriptArgs.forEach(function(arg, i) {
-      if (arg === '{runNum}') {
-        scriptArgs[i] = curRuns;
-      }
-    });
+    scriptArgs.forEach(
+      function(arg, i) {
+        switch (arg) {
```

Comment:
What's the rational behind changing the single `if` condition into a `switch` with a single `case`?